.github/workflows/test-all.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     - name: Install Go 1.19.5
```

Comment:
This can be either Install Go or Install Go 1.19.6

Reply:
Adopted.